src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -64,7 +65,9 @@ class JenkinsJobManager {
         for(ConcreteJob missingJob in missingJobs) {
             println "Creating missing job: ${missingJob.jobName} from ${missingJob.templateJob.jobName}"
             jenkinsApi.cloneJobForBranch(missingJob, templateJobs)
-            jenkinsApi.enableJob(missingJob.jobName)
+            if (!isUnitTestOnly && !missingJob.jobName.contains("Rev.com-featurebranch-deploy-feature")) {
```

Comment:
Yes, I had thought about that as well before splitting them in the proposal. The direction of keeping this in the code could use definitely use `(feature|unittest)\/.+` and then enable the branches.